netmaster/objApi/apiController.go

```diff
@@ -116,7 +116,7 @@ func NewAPIController(router *mux.Router, objdbClient objdb.API, storeURL string
 			NetworkInfraType: "default",
 			Vlans:            "1-4094",
 			Vxlans:           "1-10000",
-			FwdMode:          "bridge",
+			FwdMode:          "", // set empty fwd mode by default
```

Comment:
can be removed, variables are initialized by default

Reply:
i would leave as empty string here and in next commit to make it accept from CLI